packages/webdriverio/src/utils.ts

```diff
@@ -163,12 +163,29 @@ export const axeRunLegacy = ({
   );
 };
 
+export const openAboutBlank = (client: BrowserObject) => {
+  return promisify(
+    client.execute(`
+      window.open('about:blank', '_blank');
```

Comment:
This will fail the new test fixture test. Also need to add the test fixture test.